src/config/ConfigManager.cpp

```diff
@@ -2185,6 +2229,18 @@ std::string CConfigManager::getBoundMonitorStringForWS(const std::string& wsname
     return "";
 }
 
+const CSubmapOptions* CConfigManager::getSubmapFromName(const std::string& submap) {
+    if (submap.empty())
+        return nullptr;
+
+    CSubmapOptions* found = nullptr;
+    auto            it    = std::find_if(m_vSubmaps.begin(), m_vSubmaps.end(), [&submap](const CSubmapOptions& search) { return search.getName() == submap; });
+    if (it != m_vSubmaps.end())
+        found = &(*it);
```

Comment:
why not just `return &(*it);`? saves a line cuz you can remove `CSubmapOptions* found = nullptr;` and just `return nullptr;` at the end

Reply:
Yeah lines are expensive. I'll change it later

Reply:
lines might not be but code consistency and clarity is important. Besides, it takes 10s to fix this :)

Reply:
I would argue it's better the way I did. But it's your project